docs/architecture/shared-nothing.txt

```diff
@@ -125,39 +125,44 @@ cluster and coordinate operations:
 * Schemas of tables
 * The status and location of primary and replica shards
 
-When the Primary updates the cluster state it will publish the new state to all
+When the master node updates the cluster state it will publish the new state to all
 nodes in the cluster and and wait for all nodes to respond before processing
 the next update.
 
-.. _shared_nothing_primary_election:
+.. _shared_nothing_master_node_election:
 
-Primary Election
-----------------
+Master Node Election
+--------------------
 
-Shared nothing architecture is often also referred to as "masterless"
-architecture. For CrateDB, this is fully true in terms of data storage.
-However, coordinating metadata, settings and DDL requires to have a dynamic
-Metadata Primary to be elected. In a CrateDB cluster any node is eligible to be
-elected as a Metadata Primary, although this could also be restricted to a
-subset of nodes if required. There must be only one single Metadata Primary per
-cluster.
+The process of designating a single node as master node in a cluster is
+called Master Node Election. There must be only one single master node per cluster.
+In a CrateDB cluster any node is eligible to be elected as a master node, although
+this could also be restricted to a subset of nodes if required.
+These nodes will then elect a single node as master to coordinate the cluster state
+across the cluster.
```

Comment:
I suggest we simplify this a bit: 

"The process of designating a single node as master node in a cluster is
called Master Node Election." 
=====> "The process of electing a node as the master node in a cluster is called Master Node Election"

"There must be only one single master node per cluster" ====> "There must be only one master node per cluster at any single time" (master nodes can change, the previous sentence implied a node is forever a master node)